lib/watch-files.js

```diff
@@ -29,11 +52,12 @@ module.exports = (nameOfDirectory) => {
         const parsed = path.parse(typeFile);
         pattern = parsed.base;
         command.type = parsed.dir.replace(path.sep, '');
-      } else {
-        pattern = file.replace(`${nameOfDirectory}${path.sep}`, `*${path.sep}`);
       }
-      logResult('change', `occurred on ${file}, kicking off upload`, 'magentaBright');
```

Comment:
move this log up to when you are starting the upload process for that file.

Reply:
So no log on change, instead only when upload kicks off? Or both? I personally prefer having both, but I'm not sure how overloaded the console would get with that.

Reply:
I like having them both. 
You could also be even fancier, and do one where you "queueing upload", and a "re-queuing upload" depending if you are moving it to the back of the queue or not.